opensubtitles/common.go

```diff
@@ -0,0 +1,153 @@
+package opensubtitles
+
+import "time"
+
+//Subtitles
+type Subtitles struct {
+	TotalPages int    `json:"total_pages"`
+	TotalCount int    `json:"total_count"`
+	Page       int    `json:"page"`
+	Data       []Data `json:"data"`
+}
+//Uploader
+type Uploader struct {
+	ID   int    `json:"id"`
+	Name string `json:"name"`
+	Rank string `json:"rank"`
+}
+//FeatureDetails
+type FeatureDetails struct {
+	FeatureID   int    `json:"feature_id"`
+	FeatureType string `json:"feature_type"`
+	Year        int    `json:"year"`
+	Title       string `json:"title"`
+	MovieName   string `json:"movie_name"`
+	ImdbID      int    `json:"imdb_id"`
+	TmdbID      int    `json:"tmdb_id"`
+}
+//RelatedLinks
+type RelatedLinks struct {
+	Label  string `json:"label"`
+	URL    string `json:"url"`
+	ImgURL string `json:"img_url"`
+}
+//Files
```

Comment:
Codacy found an issue: [comment on exported type Files should be of the form "Files ..." (with optional leading article)](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)